src/mds/CDir.cc

```diff
@@ -1577,6 +1578,8 @@ void CDir::fetch(MDSContext *c, std::string_view want_dn, bool ignore_authpinnab
 
   if (mdcache->mds->logger) mdcache->mds->logger->inc(l_mds_dir_fetch);
 
+  mdcache->mds->balancer->hit_dir(this, META_POP_FETCH);
```

Comment:
How odd these were never wired up!! Great catch!

Reply:
Yep. They were defined, but they were not considered so far.